gulpfile.mjs

```diff
@@ -28,6 +30,8 @@ gulp.task('watch-css', () => {
 });
 
 gulp.task('watch-manifest', () => {
+  // Ensure build dir exists, otherwise `gulp.watch` may not work.
+  mkdirSync('build');
```

Comment:
According to the docs, this function throws if the directory already exists, unless `recursive: true` is passed, so we probably want to pass that.

https://nodejs.org/api/fs.html#fsmkdirpath-options-callback

Another option would be to check if the directory exists before trying to create it:

```js
if (!existsSync('build')) {
  mkdirSync('build');
}
```

Reply:
Oh, indeed. Using `{ recursive: true }` is better, as it avoids a potential race condition with any other processes/tasks that might try to create the directory at the same time.